tests/store_test.go

```diff
@@ -2,25 +2,36 @@ package tests
 
 import (
 	"context"
+	"crypto/tls"
+	"crypto/x509"
 	"fmt"
 	pb "github.com/aau-network-security/haaukins-store/proto"
 	"github.com/dgrijalva/jwt-go"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/credentials"
 	"google.golang.org/grpc/status"
-	"testing"
+	"io/ioutil"
 	"os"
+	"strings"
+	"testing"
 )
 
 const (
-	AUTH_KEY    = "au"
+	AUTH_KEY = "au"
+)
+
+var (
+	testCertPath    = strings.Replace(os.Getenv("CERT"), "./tests/","./",1)
+	testCertKeyPath = strings.Replace(os.Getenv("CERT_KEY"), "./tests/","./",1)
+	testCAPath 		= strings.Replace(os.Getenv("CA"), "./tests/","./",1)
```

Comment:
Since certificates are under `tests` directory, the path which is gained from `env var`should be changed to `./certs/.....`

Reply:
i am not sure i got it, can not we just the provide the path to the `.env ` file and get the content with `os.Getenv` ?

Reply:
It differs on CI :(